src/+otp/+constrainedpendulum/constraintsjacobian.m

```diff
@@ -0,0 +1,27 @@
+function dc = constraintsjacobian(~, state, g, m, ~, ~)
```

Comment:
Is this to be the standard for otp when getting the Jacobian for Half-Explicit methods? Or is this an internal functionality of the problem?

Reply:
if the problem is posed in terms of invariants, this should be standard. I might rename it to invariantJacobian